src/util/common.ts

```diff
@@ -0,0 +1,11 @@
+export function isStringArray(obj: unknown): obj is string[] {
+  return Array.isArray(obj) && obj.every(isString);
+}
+
+export function isString(obj: unknown): obj is string {
+  return typeof obj === 'string';
+}
+
+export function isObject(obj: unknown): obj is object {
+  return typeof obj === 'object';
+}
```

Comment:
Nit: Does it make sense to pull this functions into a `common` file when there is a singular usage of them? (eg, if you have a second usage you think they should go to soon, then I'm okay with this)

Reply:
My vision is for this to be used across all the files where we currently have this repetition

![image](https://user-images.githubusercontent.com/38781512/187714841-3cdef683-c1dd-485c-a659-c79f70f808a4.png)

Reply:
For `isObject`

![image](https://user-images.githubusercontent.com/38781512/187715970-9e1d0875-5557-45a5-945b-635761098489.png)

I guess the only one with singular use case is `isStringArray`. I will extract this one to the `services/policy-diff.ts` file